test/algorithms/set_operations/difference/difference.cpp

```diff
@@ -92,15 +92,13 @@ void test_all()
         1, 5, 1.0,
         1, 5, 1.0);
 
-#if defined(BOOST_GEOMETRY_USE_RESCALING) || defined(BOOST_GEOMETRY_TEST_FAILURES)
```

Comment:
This was already fine now, it is not fixed by this particular fix

Reply:
Sure, but this comment will be lost in the history of commits. Is it better to add this changes in a separate commit? Also applies to other tests in this PR.

Reply:
:+1: I'll do

Reply:
:heavy_check_mark: done, thanks